app/views/reservations/new.html.haml

```diff
@@ -40,7 +40,7 @@
           = t('reservations.new.notify')
 
   %ul.inline
-    %li= f.submit 'Create', class: 'btn', id: 'reservation_submit'
+    %li= f.submit 'Create', class: 'btn', id: 'reservation_submit', data: { disable_with: 'Create' }
```

Comment:
I wanted to keep the change small here, so I didn't go all rubocop on this file.